### DISCUSSION TITLE
eRouška VS britská NHS Covid-19

### DISCUSSION THREAD

**Discussion by kkoudelka** [covid19cz/erouska-android#522] [Ideas]
Zdravím,
když porovnávám eRoušku a britskou appku NHS Covid-19, tak celkem chápu, proč eRouška v Čechách pořádně nefunguje - a to tím, že se prezentuje jako aplikace, která nesbírá přesnou polohu uživatele. Ale k čemu je potom taková tracking appka?

**Návrh na změny:**
- používejte bluetooth + GPS pro lepší trackování (alespoň můžete dělat statistiky na daných místech)
- implementace QR kódu pro trackování specifických míst (obchody, ...)

Právě QR kódy mi přišly hrozně super, protože díky tomu budete vědět, kde se uživatelé nacházejí.
Příklad jak jsme to v UK měli - přišel jsem do knihovny, u vstupu jsem naskenoval QR kód, tj. udělal jsem check-in, díky čemuž systém věděl, že jsem skutečně v knihovně a nemusel odhadovat z GPS dat.

**Comment by davidvavra** [comment]
Zdravím, toto je spíš námět k diskuzi. Snažíme se eRoušku držet maximálně anonymní, aby nevyžadovala nic víc než musí. Díky tomu si to nainstaluje více lidí, protože se nebudou cítit sledováni. Např. lokační data jsou hodně ošemetná, lidé nechtějí dávat přístup ke své poloze. Tento názor zakladáme na popularitě eRoušky 1.0 vs 2.0. 1.0 byla méně anonymní, registrovalo se přes telefonní číslo a data se posílaly hygieně. 2.0 je plně anonymní, data se neposílají hygieně ale jenom mezi uživateli. A eRouška 2.0 má několikanásobně více uživatelů. Nápad s checkováním pomocí QR kódů se mi líbí, ale možná by to chtěl mít v jiné aplikaci - aby nešlo nijak spojit data z anonymní eRoušky s těmi citlivými.
